core/mo-services-xml/src/main/resources/xml/ServiceDefPLATFORM.xml

```diff
@@ -1732,9 +1746,21 @@
           <mal:item comment="Reaction wheel w"
                     nvalue="6" value="WHEEL_W" />
         </mal:enumeration>
+        <mal:enumeration
+          comment="Attitude determination mode identifier."
+          name="AttitudeDeterminationMode" shortFormPart="13">
+          <mal:item comment="Attitude determination using Magnetic and Sun during pre-alignment"
+                    nvalue="1" value="MGT_SUN" />
+          <mal:item comment="Attitude determination using StarTracker with Magnetic and Sun"
+                    nvalue="2" value="STRTRK_MGT_SUN" />
+          <mal:item comment="Attitude determiantion using StartTracker only"
+                    nvalue="3" value="STRTRK_ONLY" />
+          <mal:item comment="Attitude determination invalid"
+                    nvalue="4" value="Invalid" />
+        </mal:enumeration>
         <mal:composite
           comment="Holds Raction wheel parameters" name="ReactionWheelParameters"
```

Comment:
Please fix a typo while at it. `Raction` -> `reaction`

Reply:
fixed